### ISSUE TITLE
NPE in Event Handler - onEntitySpawnCheck(EventHandler.java:216)

### ISSUE THREAD

**Issue by Lathanael** [BluSunrize/ImmersiveEngineering#630]
Crash excerpt:
```
Description: Exception ticking world

java.lang.NullPointerException: Exception ticking world
	at blusunrize.immersiveengineering.common.EventHandler.onEntitySpawnCheck(EventHandler.java:216)
	at cpw.mods.fml.common.eventhandler.ASMEventHandler_475_EventHandler_onEntitySpawnCheck_CheckSpawn.invoke(.dynamic)
	at cpw.mods.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:54)
	at cpw.mods.fml.common.eventhandler.EventBus.post(EventBus.java:138)
	at net.minecraftforge.event.ForgeEventFactory.canEntitySpawn(ForgeEventFactory.java:112)
	at net.minecraft.world.SpawnerAnimals.func_77192_a(SpawnerAnimals.java:169)
	at net.minecraft.world.WorldServer.func_72835_b(WorldServer.java:161)
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:625)
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:547)
	at net.minecraft.server.integrated.IntegratedServer.func_71217_p(IntegratedServer.java:111)
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:396)
	at net.minecraft.server.MinecraftServer$2.run(MinecraftServer.java:685)
```
I believe this is related to #587. The game crashes right after/while loading.
Getting this crash with IE 0.6.1, BC 7.1.13, BCC 7.1.2 and IM 59.1.1. (all latest to my knowledge)

After turning the BC compat module for immibis microblocks off the world loads fine.

Full crash log: https://gist.github.com/Lathanael/7400d404dfeb4e1bc924

**Comment by theunkn0wn1** [comment]
Yeah. Known issue.

Duplicate of #587.


You need not create duplicate issues, it just adds confusion to the mix. Last I checked, the fix for your issue is in the works

**Comment by Lathanael** [comment]
Sorry for the duplicate. Somehow i was sure that the NPE was fixed in 0.6.1 and wanted to report it wasn't. But upon checking the commit log today i realized that it was PRed after the 0.6.1 release.

Really sorry!
